### ISSUE TITLE
Preview mode assessments are logging errors

### ISSUE THREAD

**Issue by zachberry** [ucfopen/Obojobo#658]
Happening on dev/4 - Submit an assessment attempt in preview mode and this gets logged:

```
[0] Wed, 09 Jan 2019 15:41:01 GMT obojobo_server:info  LTI begin sendHighestAssessmentScore for userId:"1", draftId:"cd3091ea-bca4-49bd-ab7c-8d44a53dd49d", assessmentId:"my-assessment" f59d6899-dca5-41d6-b941-94958a9ff4b5
[0] Wed, 09 Jan 2019 15:41:01 GMT obojobo_server:error LTI no assessment score found, unable to proceed! f59d6899-dca5-41d6-b941-94958a9ff4b5
[0] Wed, 09 Jan 2019 15:41:01 GMT obojobo_server:info  LTI gradebook status is "error_state_unknown" f59d6899-dca5-41d6-b941-94958a9ff4b5
[0] Wed, 09 Jan 2019 15:41:01 GMT obojobo_server:info  LTI store "error_no_assessment_score_found" success - id:"9" f59d6899-dca5-41d6-b941-94958a9ff4b5
[0] Wed, 09 Jan 2019 15:41:01 GMT obojobo_server:info  LTI complete f59d6899-dca5-41d6-b941-94958a9ff4b5
[0] Wed, 09 Jan 2019 15:41:01 GMT obojobo_server:info  End attempt "67d5c82b-73f4-4a53-bc5b-641be8ef7e91" - sendLTIScore was executed
```

That shouldn't be happening - it should be aware that it's in preview mode and store the 'preview' states. Need to check if this is happening on master and see what changed.

**Comment by zachberry** [comment]
Tested, is working as expected on master

**Comment by gitnix** [comment]
addressed in #659
